ISX/ISX/CabinCrewViewController.swift

```diff
@@ -47,6 +50,55 @@ class CabinCrewViewController: UITableViewController {
         })
     }
     
+    func observerNewRequest() {
+        requestsRef?.queryLimited(toLast: 1).observe(.childAdded, with: { snapshot in
+            let latestRequest = Request(snapshot: snapshot)
+            let customerChair = latestRequest.customerChair
+            self.localNot(customerChair)
+            self.tableView.reloadData()
+        })
+    }
+    
+    func localNot(_ customerChair: String) {
```

Comment:
function names verbNoun(Reason) something like sendLocalNotification, but since this method seems to check authorization and then either requests authorization or schedules a local notification -> checkAuthStatusProceed or something else